lib/kdump_utils.pm

```diff
@@ -327,6 +327,8 @@ sub check_function {
     assert_script_run 'find /var/crash/';
 
     if ($args{test_type} eq 'function') {
+        # Check, that vmcore exists, otherwise fail
+        assert_script_run('ls /var/crash/*/vmcore');
```

Comment:
Just a  sugar thingy, maybe `test -f` would be more straightforward. But that is only neat-picking, feel free to ignore.

Reply:
Or use `ls -lah` to see the size (can be good diagnostic)

Reply:
@pevik This is even better, thx.

Reply:
Implemented with `ls -lah` and re-done verification runs.